camel/source/pkg/apis/sources/v1alpha1/camelsource_types.go

```diff
@@ -137,6 +138,22 @@ func (s *CamelSourceStatus) MarkSink(uri string) {
 	}
 }
 
+// MarkSinkWarnDeprecated sets the condition that the source has a sink configured and warns ref is deprecated.
+func (s *CamelSourceStatus) MarkSinkWarnRefDeprecated(uri string) {
+	s.SinkURI = uri
+	if len(uri) > 0 {
+		c := apis.Condition{
+			Type:     CamelConditionSinkProvided,
+			Status:   corev1.ConditionTrue,
+			Severity: apis.ConditionSeverityError,
+			Message:  "Using deprecated object ref fields when specifying spec.sink. Update to spec.sink.ref. These will be removed in 0.11.",
```

Comment:
this feels strange as the next release is 1.11. Maybe replace by "These will be removed in a future release"

Reply:
Right, I've fixed the message also in the other sources that were printing the same info.